ext/pycall/pycall.c

```diff
@@ -666,6 +666,7 @@ pycall_libpython_api_PyList_GetItem(VALUE mod, VALUE pyptr, VALUE idx)
   if (!pyobj_item) {
     pycall_pyerror_fetch_and_raise("PyList_GetItem in pycall_libpython_api_PyList_GetItem");
   }
+  pycall_Py_DecRef(pyobj_item);
```

Comment:
I've noticed that we should call `IncRef` here for `pyobj_item` because `PyList_GetItem` doesn't change the reference count of the returned object.

At least inserting `DecRef` here should be wrong.

We didn't have trouble coincidentally because`pycall_libpython_api_PyList_GetItem` isn't called from elsewhere.  I'll remove this function later.

So could you please remove this line?

Reply:
I see, I will delete it.